### ISSUE TITLE
LAS file converted with Coverter v2.0.1 does not work as expected in Potree Viewer

### ISSUE THREAD

**Issue by htejwani** [potree/PotreeConverter#441]
https://geoslam.com/Potree/Potree/harbor/files/harbor_las.zip

I converted the above  LAS file with Potree converter 2.0.1. The model is rendered blurry and freezes the page when trying to zoom-in in the Potree Viewer. When converted with v1.7, it works without any issues.

**Comment by potree** [comment]
Hi,

The data sets look the same to me in Potree and in Cloud Compare. Adaptive point sizes aren't ideal for this data set, so you may prefer setting the point sizing to fixed, and perhaps to 1 px:

```
material.size = 1;
material.minSize = 1;
material.pointSizeType = Potree.PointSizeType.FIXED;
```

Potree |   Cloud Compare
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/2094934/91690542-6b714b80-eb66-11ea-9c4a-40892259098a.png) |  ![image](https://user-images.githubusercontent.com/2094934/91690550-6dd3a580-eb66-11ea-9eb9-3e2a652caea3.png)


Not sure where the freezing is coming from, however. Does it always happen? Does it happen in certain regions? Is there a way to reliably reproduce it so that I could debug it? Can you try with your browsers developer tools activated (ctrl + shift + i or F12) and check for error messages in the console?

**Comment by htejwani** [comment]
Yes, the freezing always happens, I am using chrome and it occurs in the Desktop app too. I do not see any errors in the console when it freezes.  But this one error comes up intermittently: Failed to load resource: net::ERR_CACHE_OPERATION_NOT_SUPPORTED for this url: http://localhost:1234/pointclouds/harbor_las2_converted/octree.bin
 
Potree.PointSizeType.FIXED did not make much of a difference.

**Comment by potree** [comment]
> Failed to load resource: net::ERR_CACHE_OPERATION_NOT_SUPPORTED

That's normal in chrome and will trigger a second try to load the data. Do you see anything at all or does it already freeze before the first part of the data is rendered? Can you try in Firefox?

What OS and browser/browser versions are you using?

**Comment by potree** [comment]
Also, check the "network" tab in the chrome developer tools to see if there is something strange going on, like a data set loading forever

**Comment by htejwani** [comment]
OS: Windows 10, Chrome v84, sorry can't install firefox, not allowed in my company. It freezes after I zoom-in 2-3 levels and then it stops zooming-in further. Network tab has a bunch of request with 206 response codes. No it doesn't indicate that it's trying to load something forever.
I see this error 'failed to load r2426'  (OcTreeLoader_1_8.js: 120) and it says trying again! after the cache error. That is all I see.

**Comment by htejwani** [comment]
Once the zooming freezes and I double tap on my touch pad, it zooms in to the highest and the zoom out and zoom-in starts working from there but the rendering (FIXED or Adaptive) is still all over the place. It's nowhere as good as the one with 1.7.

**Comment by m-schuetz** [comment]
Could you send me a zip of the converted files? I can't reproduce the error on my end.

**Comment by htejwani** [comment]
Do not have any means to share that big a file I guess, it's about a GB in size.  Snapshot of the 3 files and their sizes here:
![image](https://user-images.githubusercontent.com/5311510/91701257-a4281980-eb94-11ea-80ad-5c5056e3eed6.png)
 
Also, after increasing the point budget, the rendering is much better, this is something that I missed but the zoom in freezing issue until I double tap remains.

**Comment by midnight-dev** [comment]
@htejwani A free Google account will let you upload and share files up to 10 GB in size. You can also compress the point cloud data with something like 7zip (LZMA or BZip2 archive types) could get it under a gigabyte if you need to get file sizes under 1 GB. In fact, with archiving software like 7zip, you can split up files into specifically-sized chunks, i.e. you can set a max size to a gigabyte and if the compression can't get it below, another file will be created. Put the files in the same place and any compatible program will decompress the contents as expected without issue. 

Dropbox should also offer plenty of space to share with a free account, but I haven't looked at their limitations in a long time.

**Comment by htejwani** [comment]
Sure, thanks. Here's the link:

https://drive.google.com/drive/folders/107lv7QFwUXssEogEHajqo3T8ALGm6hAf

**Comment by htejwani** [comment]
Hi @m-schuetz, did you get a chance to look and this and reproduce this issue?
